.pylintrc

```diff
@@ -419,9 +419,7 @@ redefining-builtins-modules=six.moves,past.builtins,future.builtins,builtins,io
 
 
 # List of note tags to take in consideration, separated by a comma.
-notes=FIXME,
-      XXX,
-      TODO
```

Comment:
Oops this needs to stay here otherwise pylint won' notify you if there is TODO, but in the disable section the `fix-me` needs to be removed.